tbx/project_styleguide/templates/patterns/pages/team/team_listing.html

```diff
@@ -17,11 +17,11 @@
                 <li class="team-listing__item">
                     {# lazy load after the first five #}
                     {% if forloop.counter > 5 %}
-                        {% srcset_image member.image format-webp fill-{230x230,370x370} sizes="(max-width: 598px) 370px, (min-width: 599px) 230px" class="team-listing__image" alt="" loading="lazy" %}
+                        {% srcset_image member.image fill-{230x230,370x370} format-webp sizes="(max-width: 598px) 370px, (min-width: 599px) 230px" class="team-listing__image" alt="" loading="lazy" %}
```

Comment:
What was the logic for swapping the order here? Is it to match the prefetching?

Reply:
pretty much. "fill-230x230|format-web" reads nicer than "format-webp|fill-230x230" (at least in my head)